src/typings/index.ts

```diff
@@ -10,6 +9,7 @@
 /// <reference path="platformsFund.ts" />
 /// <reference path="platformsNotificationConfiguration.ts" />
 
+export * as binlookip from './binlookup/models';
```

Comment:
Typo:
```suggestion
export * as binlookup from './binlookup/models';
```

Reply:
Good catch! we should maybe change. our unit tests to use this class in order to test the names.